demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -171,6 +172,42 @@ def is_image_path_valid(self) -> bool:
             return False
         return True
 
+    def verify_no_empty_sections(self) -> bool:
+        """ Check the following:
+            1. if Troubleshooting exists, it isn't empty.
+            2. no 'FILL IN REQUIRED PERMISSIONS HERE'.
+            3. no unexplicit version number - such as "version xx of".
+
+        Returns:
+            bool: True If all req ok else False
+        """
+        with open(self.file_path) as f:
+            is_valid = True
+            errors = ""
+            readme_content = f.read()
+            troubleshooting = re.findall(r'(## Troubleshooting\n\n?)(.*)', readme_content, re.IGNORECASE)
```

Comment:
can there be any other empty section? I see also `limitations` and `use cases`
BTW, probably these 3 sections should be in the same validation func.

Reply:
sounds good I'll check for more sections and will add them to the issue too.